library-no-op/build.gradle

```diff
@@ -19,6 +20,7 @@ artifacts {
 
 dependencies {
     implementation "com.squareup.okhttp3:okhttp:$okhttp3Version"
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
```

Comment:
If we merge this PR after #40, $kotlin_version should become $kotlinVersion.

Reply:
Done ✅